client/verta/tests/test_experimentrun/test_deployment.py

```diff
@@ -1,318 +0,0 @@
-            headers=conn.auth,
-        )
-
-    def test_already_undeployed_undeploy(self, experiment_run):
-        # should not raise error
-        experiment_run.undeploy()
-
-
-@pytest.mark.skip(reason="old deployment API is being phased out (VR-7935)")
-class TestGetDeployedModel:
-    def test_get(self, experiment_run, model_for_deployment):
-        model = model_for_deployment['model'].fit(
-            model_for_deployment['train_features'],
-            model_for_deployment['train_targets'],
-        )
-
-        experiment_run.log_model(model, custom_modules=[])
-        experiment_run.log_environment(Python(['scikit-learn']))
-
-        experiment_run.deploy(wait=True)
-
-        deployed_model = experiment_run.get_deployed_model()
-        x = model_for_deployment['train_features'].iloc[1].values
-        deployed_model.predict([x])
-
-        deployed_model_curl = deployed_model.get_curl()
-        deployed_status = experiment_run.get_deployment_status()
-        assert deployed_status["url"] in deployed_model_curl
-        assert deployed_status["token"] in deployed_model_curl
-
-        conn = experiment_run._conn
-        requests.delete(
-            "{}://{}/api/v1/deployment/models/{}".format(conn.scheme, conn.socket, experiment_run.id),
-            headers=conn.auth,
-        )
-
-    def test_not_deployed_get_error(self, experiment_run, model_for_deployment):
-        with pytest.raises(RuntimeError):
-            experiment_run.get_deployed_model()
-
-    def test_undeployed_get_error(self, experiment_run, model_for_deployment):
-        experiment_run.log_model(model_for_deployment['model'], custom_modules=[])
-        experiment_run.log_environment(Python(['scikit-learn']))
-
-        experiment_run.deploy(wait=True)
-        experiment_run.undeploy(wait=True)
-
-        with pytest.raises(RuntimeError):
-            experiment_run.get_deployed_model()
-
-        conn = experiment_run._conn
-        requests.delete(
-            "{}://{}/api/v1/deployment/models/{}".format(conn.scheme, conn.socket, experiment_run.id),
-            headers=conn.auth,
-        )
-
-
-class TestGitOps:
-    def test_download_deployment_yaml(self, experiment_run, model_for_deployment, in_tempdir):
-        download_to_path = "deployment.yaml"
-
-        experiment_run.log_model(
-            model_for_deployment['model'],
-            custom_modules=[],
-            model_api=model_for_deployment['model_api'],
-        )
-        experiment_run.log_environment(Python(['scikit-learn']))
-
-        filepath = experiment_run.download_deployment_yaml(download_to_path)
-        assert filepath == os.path.abspath(download_to_path)
-
-        # can be loaded as YAML
-        with open(filepath, 'rb') as f:
-            model_deployment = yaml.safe_load(f)
-
-        assert model_deployment['kind'] == "ModelDeployment"
-        assert model_deployment['metadata']['name'] == experiment_run.id
```

Comment:
We should keep this (unless `download_deployment_yaml()` is also defunct)! Everything else can go.

Reply:
Is [`download_endpoint_manifest`](https://verta.readthedocs.io/en/master/_autogen/verta.Client.html?highlight=download_endpoint_manifest#verta.Client.download_endpoint_manifest) its replacement??

Reply:
Yes